CTP/shell/src/com/navercorp/cubridqa/shell/common/Constants.java

```diff
@@ -175,7 +175,7 @@ private static ShellScriptInput createLinKillScripts (boolean inLocal){
 		scripts.addCommand(bothKill("ps -u $USER -o pid,comm| grep -v grep | grep shard | awk '{print $1}'"));
 		scripts.addCommand("ipcs | grep $USER | awk '{print $2}'  | xargs -i ipcrm -m {}");
 		if (inLocal == false) {
-			scripts.addCommand(bothKill("ps -u $USER -o pid,comm| grep -v grep | grep -i java | awk '{print $1}'"));
+			scripts.addCommand(bothKill("ps -u $USER -o pid,command| grep -v grep | grep -i java |grep -v '" + com.navercorp.cubridqa.common.Constants.CTP_PACKAGE_NAME + "' | awk '{print $1}'"));
```

Comment:
Try to use this way:
ctp_java_pid_list=`ps -u $USER -o pid,command| grep -v grep | grep -E "com.navercorp.cubridqa|service.Server" | awk '{print $1}'`;
Then kill all java which is not in ${ctp_java_pid_list}.